Chary/Assignments/Functions/1_vowels_uppercase.py

```diff
@@ -0,0 +1,20 @@
+#Write a program convert all the vowels to uppercase for the given input
+#string.
+
+def vowels_uppercase(x):
+    y=[]
+    vowels="aeiou"
+    for i in x:
```

Comment:
Please provide meaning full variable names for better readability  
For example   ch instead of i